packages/a/aws-sdk-cpp/xmake.lua

```diff
@@ -0,0 +1,30 @@
+package("aws-sdk-cpp")
+    set_homepage("https://github.com/aws/aws-sdk-cpp")
+    set_description("AWS SDK for C++")
+
+    add_urls("https://github.com/aws/aws-sdk-cpp.git")
+    add_versions("1.9.333", "6ec095fa11379174da20bf1523c93c92503eb8e2")
+
+    add_configs("build_only", {description = 'By default, all SDKS are built, if only AWS S3 is required, then set build_only="s3", with multiple SDKS separated by commas.'})
+    add_deps("cmake")
+
```

Comment:
add_deps("curl")

Reply:
```shell
error: @programdir/core/main.lua:280: @programdir/modules/private/action/require/impl/package.lua:758: package(curl) not found!
```

Reply:
it's actually called `libcurl`, `add_deps("libcurl")`

Reply:
> it's actually called `libcurl`, `add_deps("libcurl")`

Thx